proto/ibc/applications/fee/v1/tx.proto

```diff
@@ -48,6 +55,24 @@ message MsgRegisterCounterpartyAddress {
 // MsgRegisterCounterpartyAddressResponse defines the response type for the RegisterCounterpartyAddress rpc
 message MsgRegisterCounterpartyAddressResponse {}
 
+// MsgRegisterDistributionAddress defines the request type for the RegisterDistributionAddress rpc
+message MsgRegisterDistributionAddress {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // the relayer address
+  string address = 1;
```

Comment:
rename to `relayer_address`?

Reply:
Yep, will do this in a follow up PR tomorrow to reduce the number of changes! We will rename to `RegisterCounterpartyPayee` and align the same field names..etc 👍